docker_compose_depends_on.md

```diff
@@ -3,7 +3,7 @@
 In Docker Compose, depends_on can only control the start and stop of containers in dependency order. However, for startup Compose does not wait until a container is “ready” (whatever that means for your particular application). Docker published this [best practice for controlling startup and shutdown order in Compose](https://docs.docker.com/compose/startup-order/).
 
 # App Service Multi-container Web App Best Practice 
-[App Service Multi-container (Preview)](https://blogs.msdn.microsoft.com/appserviceteam/2018/05/07/multi-container/) doesn’t support the depends_on option in Docker Compose configuration, we would ignore the option if customer specifies it in the Docker-Compose yaml file. Similar to what is recommended by Docker, we recommend App Service customers to handle the dependencies by performing the check in your application code, both at startup and whenever a connection is lost for any reason. Design your application to attempt to re-establish a connection to the database after a failure. If the application retries the connection, it can eventually connect to the database. 
+[App Service multi-container (preview)](https://docs.microsoft.com/en-us/azure/app-service/containers/tutorial-multi-container-app) doesn’t support the depends_on option in Docker Compose configuration, we would ignore the option if customer specifies it in the Docker-Compose yaml file. Similar to what is recommended by Docker, we recommend App Service customers to handle the dependencies by performing the check in your application code, both at startup and whenever a connection is lost for any reason. Design your application to attempt to re-establish a connection to the database after a failure. If the application retries the connection, it can eventually connect to the database. 
```

Comment:
@dorfire can you update the PR to target this link: 
https://docs.microsoft.com/azure/app-service/containers/tutorial-multi-container-app

Removing the /en-us/ segment from the URL ensures folks land on localized content when present.